crates/client/src/lib.rs

```diff
@@ -86,23 +86,27 @@ pub async fn connect(
 
     let path = path.as_ref().to_path_buf();
 
-    // Taken from https://github.com/hyperium/tonic/blob/eeb3268f71ae5d1107c937392389db63d8f721fb/examples/src/uds/client.rs#L19
+    // Taken from https://github.com/hyperium/tonic/blob/71fca362d7ffbb230547f23b3f2fb75c414063a8/examples/src/uds/client.rs#L21-L28
     // There will ignore this uri because uds do not use it
     // and make connection with UnixStream::connect.
-    let channel = Endpoint::try_from("http://[::]")
-        .unwrap()
+    let channel = Endpoint::try_from("http://[::]")?
         .connect_with_connector(tower::service_fn(move |_| {
-            #[cfg(unix)]
-            {
-                tokio::net::UnixStream::connect(path.clone())
-            }
-
-            #[cfg(windows)]
-            {
-                let client = tokio::net::windows::named_pipe::ClientOptions::new()
-                    .open(path.clone())
-                    .map_err(|e| std::io::Error::from(e));
-                async move { client }
```

Comment:
does this not work with the latest versions?

Reply:
No - see latest commit/checks. I don't have a windows machine so I'm just guessing and checking from CI 🙈

Reply:
got it :-), I've added a suggestion that is building on my windows machine